Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -505,13 +505,6 @@ - (CGSize)sizeThatFits:(CGSize)size
   return fittingSize;
 }
 
-#pragma mark - Accessibility
```

Comment:
I am afraid this particular change is simply wrong/unreasonable.

Reply:
Just to be clear: Removing this turns off all accessibility props of <TextInput> (by redirecting them to the wrapper view).

Reply:
@shergin my idea of deleting this `reactAccessibilityElement` was to make `RCTBaseTextInputView` use [`reactAccessibilityElement` inherited from `RCTView`](https://github.com/facebook/react-native/blob/master/React/Views/RCTView.m#L316-L319) (the wrapper).

When I opened this PR I tested `TextField` passing those events set from [RCTViewManager](https://github.com/facebook/react-native/blob/master/React/Views/RCTViewManager.m#L125-L136), and it was working. But I might have missed something :-/.

Now I'll need to make more tests, especially because I this commit needed to be reverted (probably because of this detox failure https://circleci.com/gh/facebook/react-native/88586, right @grabbou?). I just don't know yet why this commit caused a regression into `Switch` component.

Reply:
Yes, I think Detox couldn't find a TextInput on the top of the app (by its accessibility label), to type `<switch />` and select that example.

So definitely related.